services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluTulos.kt

```diff
@@ -0,0 +1,22 @@
+package fi.hel.haitaton.hanke.tormaystarkastelu
+
+class TormaystarkasteluTulos {
```

Comment:
Why not using data classes here? It could have only `hankeId` (and `hankeTunnus`) as constructor parameter?